probe_scraper/scrapers/release_calendar.py

```diff
@@ -0,0 +1,105 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from collections import defaultdict
+from datetime import datetime
+
+from bs4 import BeautifulSoup
+import requests
+
+from probe_scraper.parsers.utils import get_major_version
+
+RELEASE_CALENDAR_URL = ("https://wiki.mozilla.org/api.php"
+                        "?action=parse&format=json&page=Release_Management/Calendar")
+
+
+def get_version_table_dates(table):
```

Comment:
Some info here: https://mozilla.slack.com/archives/C4D5ZA91B/p1576163365064000

I'm not necessarily sure that it's a better alternative, but we could use https://calendar.google.com/calendar/embed?src=bW96aWxsYS5jb21fZGJxODRhbnI5aTh0Y25taGFiYXRzdHY1Y29AZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ

Reply:
Parsing through the calendar seems even messier so I'll stick with this for now and maybe revisit one day

Reply:
That seems fine, can you file an issue for it?